src/MariaBinding.h

```diff
@@ -16,17 +17,17 @@ class MariaBinding : public boost::noncopyable {
   std::vector<MariaFieldType> types;
   std::vector<MYSQL_TIME> time_buffers;
 
-public:
+ public:
```

Comment:
Single-space indent for access modifiers looks awkward, but I'm willing to be convinced otherwise.